CONTRIBUTING.md

```diff
@@ -154,7 +154,7 @@ In default test mode, tests are replayed from existing recordings. In live mode,
 
 To execute the `py.test` suite using the live tests mode, execute py.test with the environment variable `LEXICON_LIVE_TESTS` set to `true` like below:
 
-	LEXICON_LIVE_TESTS=true py.test tests/providers/test_foo.py
+	LEXICON_LIVE_TESTS=true py.test lexicon/tests/providers/test_foo.py
```

Comment:
Thanks for the fix in the documentation!

Reply:
Any time! 😊